pkg/commands/root.go

```diff
@@ -1,160 +1,168 @@
 package commands
 
 import (
+	"errors"
 	"fmt"
 	"os"
-	"runtime"
-	"runtime/pprof"
-	"runtime/trace"
-	"strconv"
+	"slices"
 
+	"github.com/fatih/color"
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
 
 	"github.com/golangci/golangci-lint/pkg/config"
-	"github.com/golangci/golangci-lint/pkg/exitcodes"
 	"github.com/golangci/golangci-lint/pkg/logutils"
+	"github.com/golangci/golangci-lint/pkg/report"
 )
 
-const (
-	// envHelpRun value: "1".
-	envHelpRun        = "HELP_RUN"
-	envMemProfileRate = "GL_MEM_PROFILE_RATE"
-)
+func Execute(info BuildInfo) error {
+	return newRootCommand(info).Execute()
+}
+
+type rootOptions struct {
+	PrintVersion bool
+
+	Verbose bool   // Persistent flag.
```

Comment:
What does persistent mean here? That it is in affect for all sub commands as well?